test/itemlist_formaction.cpp

```diff
@@ -0,0 +1,154 @@
+#include "catch.hpp"
+
+#include "cache.h"
+#include "itemlist_formaction.h"
+#include "keymap.h"
+#include "stfl/itemlist.h"
+
+#include <unistd.h>
+
+using namespace newsbeuter;
+
+TEST_CASE("itemlist_formaction::process_op()") {
+
+	controller c;
+	newsbeuter::view v(&c);
+	std::string test_browser_file = "/tmp/test_browser.txt";
+	std::string test_url = "http://test_url";
+	std::string cache_path = "test-cache.db";
+	std::string line;
+	int itemCount = 6;
+	int openedItemsCount = 0;
+	int lineCount = 0;
+	int maxItems = 4;
+	int openedItemsTarget = 0;
```

Comment:
I have two comments regarding all of these `int`s:
1. Some of them look like constants, but I'm not entirely sure. If they are, just mark them `const`.
2. It's kinda hard to read the tests because I have to flick between initializations here and the test cases. I suggest to violate DRY principle and re-declare all of these (save for constants, if any) inside test cases.